<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="VI209570" ignored="0" name="test1" tests="2" failures="1" timestamp="2020-05-30T18:19:28 IST" time="0.083" errors="0">
  <testcase name="Login" time="0.02" classname="listeners.TestCases"/>
  <testcase name="TestToFail" time="0.02" classname="listeners.TestCases">
    <failure type="java.lang.AssertionError" message="did not expect to find [true] but found [false]">
      <![CDATA[java.lang.AssertionError: did not expect to find [true] but found [false]
at listeners.TestCases.TestToFail(TestCases.java:21)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- TestToFail -->
</testsuite> <!-- test1 -->
